PROJECT

```diff
@@ -18,20 +18,27 @@ resources:
   version: v1alpha1
 - api:
     crdVersion: v1
-    namespaced: false
```

Comment:
Is this removal intentional?

Reply:
I too noticed this. This change was done by operator-sdk based on its defaults I believe.